src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Microsoft.Health.Fhir.TemplateManagement.UnitTests.csproj

```diff
@@ -25,9 +25,6 @@
     <None Include="..\..\data\Templates\Hl7v2\VXU_V04.liquid" Link="TestData\UserFolder\VXU_V04.liquid">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="..\Microsoft.Health.Fhir.TemplateManagement.FunctionalTests\DefaultTemplates.tar.gz" Link="DefaultTemplates.tar.gz">
```

Comment:
There is another PR #120 solving the same issus. Do we still need that PR?

Reply:
Yes, I will close PR#120